internal/databases/postgres/configure.go

```diff
@@ -1,15 +1,49 @@
 package postgres
 
 import (
+	"fmt"
 	"time"
 
 	"github.com/pkg/errors"
 	"github.com/spf13/viper"
 	"github.com/wal-g/tracelog"
 	"github.com/wal-g/wal-g/internal"
 	"github.com/wal-g/wal-g/internal/fsutil"
+	"github.com/wal-g/wal-g/internal/multistorage"
+	"github.com/wal-g/wal-g/internal/multistorage/cache"
+	"github.com/wal-g/wal-g/pkg/storages/storage"
 )
 
+// ConfigureMultiStorageFolder configures the primary storage folder and all the failover ones, if any, and builds a
+// multi-storage folder that aggregates them. It also sets up a cache to keep storage alive check results there.
+// This function doesn't set any specific multi-storage folder policies, so policies.Default are used initially.
+func ConfigureMultiStorageFolder() (storage.Folder, error) {
```

Comment:
I moved this function as well as [`PrepareMultiStorageWalUploader`](https://github.com/wal-g/wal-g/pull/1550/files#diff-b9064099fa73c81cd39e3b8987ce68bad01730259e32d36c9100ae43e81bd5a9R75) and [`PrepareMultiStorageFolderReader`](https://github.com/wal-g/wal-g/pull/1550/files#diff-8bbbe6ade21b38b228c1d7d55c1f661da67bb808ff3fa119763e0a291ef038b6R28) from `cmd` package to here to allow the daemon to use them in its handlers (it would be bad if the daemon handlers depended on `cmd`).

But after the changes requested by @usernamedt, these functions aren't used in daemon handlers anymore.

However, I can leave them here anyway, because we may need it in the daemon again in the future, maybe for some commands other than `wal-fetch` and `wal-push`.